src/main/java/steps/LoginPageSteps.java

```diff
@@ -2,12 +2,13 @@
 
 import base.BasePage;
 import cucumber.api.java.en.*;
-
-import static constant.LoginPageConstant.SI_INPUT_PSWD_CSS_SELECTOR;
-import static constant.LoginPageConstant.SI_INPUT_UNAME_CSS_SELECTOR;
+import org.openqa.selenium.By;
 
 public class LoginPageSteps extends BasePage {
 
+    private static final By SI_INPUT_UNAME_CSS_SELECTOR = By.xpath("//*[@id=\"si_username\"]");
+    private static final By SI_INPUT_PSWD_CSS_SELECTOR = By.xpath("//*[@id=\"si_password\"]");
+
```

Comment:
I closed to PR, but constant name should include `xpath` instead of `css selector` (i.e. `SI_USERNAME_XPATH`). I am dropping note here for just collecting future refactor issues.